managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1554,13 +1554,13 @@ public void operationComplete(Void v, Stat stat) {
                         log.debug("[{}] Updating of ledgers list after create complete. version={}", name, stat);
                     }
                     ledgersStat = stat;
-                    ledgers.put(lh.getId(), newLedger);
-                    currentLedger = lh;
-                    currentLedgerEntries = 0;
-                    currentLedgerSize = 0;
                     metadataMutex.unlock();
```

Comment:
>This lock is held while the ledgers list or propertiesMap is updated asynchronously on the metadata store. Since we use the store version, we cannot have multiple concurrent updates.

Should we move this operation behind the updated memory status?

Reply:
which line ?

Reply:
@Technoboy- 

it is `metadataMutex.unlock();`

@mattisonchao 

Already moved